### ISSUE TITLE
Running treetime with sequence_length instead of alignment yields an error

### ISSUE THREAD

**Issue by theosanderson** [neherlab/treetime#169]
Hello!

```treetime --tree data/ebola/ebola.nwk --dates data/ebola/ebola.metadata.csv --sequence-length 20000```

gives 

```
23.10	###TreeTime.run: rerunning timetree after rerooting
Traceback (most recent call last):
  File "/usr/local/bin/treetime", line 12, in <module>
    return_code = params.func(params)
  File "/usr/local/lib/python3.7/dist-packages/treetime/argument_parser.py", line 213, in toplevel
    timetree(params)
  File "/usr/local/lib/python3.7/dist-packages/treetime/wrappers.py", line 567, in timetree
    tracelog_file=os.path.join(outdir, "trace_run.log"))
  File "/usr/local/lib/python3.7/dist-packages/treetime/treetime.py", line 198, in run
    sequence_marginal = seq_kwargs['marginal_sequences'], Tc=None, tracelog=tracelog_file))
  File "/usr/local/lib/python3.7/dist-packages/treetime/treetime.py", line 828, in tracelog_run
    'seq_LH'     : self.tree.sequence_marginal_LH if sequence_marginal else self.tree.sequence_joint_LH,
AttributeError: 'Tree' object has no attribute 'sequence_joint_LH'
```

I'm not certain if this is expected or not -- please do close this if it is!

Repro: https://colab.research.google.com/drive/1nMXtZOS9uQjfRwFKtQLHmJlA1y2zhV2Y?usp=sharing

Thanks!

**Comment by rneher** [comment]
thanks, Theo. I'll push a fix.

**Comment by rneher** [comment]
I released 0.8.5. Your example seems to be working now. Thanks for the great test!

**Comment by theosanderson** [comment]
Thanks @rneher !
